static/src/javascripts/bootstraps/sport.js

```diff
@@ -57,6 +59,31 @@ define([
                         $('.js-football-tabs').append(nav);
                     }
                 });
+
+                var content = $.create(resp.scoreEvents);
+                var dropdownTemplate = detect.isBreakpoint({ max: 'mobile' }) && resp.dropdown;
+                var name = 'Score breakdown';
+                var button;
+
+                if (detect.isBreakpoint({ max: 'mobile' })) {
+                    $('.js-after-article').append(
+                        $.create(dropdownTemplate).each(function (dropdown) {
+                            var $dropdown = $(dropdown).addClass('dropdown--key-events');
+                            if (config.page.isLiveBlog) { $dropdown.addClass('dropdown--key-events'); }
+                            $dropdown.addClass('dropdown--active');
+                            $('.dropdown__label', dropdown).append(name);
+                            $('.dropdown__content', dropdown).append(content);
+                            button = $('.dropdown__button', dropdown);
+                            button
+                                .attr('data-link-name', 'Show dropdown: ' + name);
+                        })
+                    );
+                } else {
+                    var placeholder = $.create('<div class="hide-on-mobile"></div>');
+                    $('.score-container').after(placeholder);
+                    placeholder.append(content);
+                }
+
```

Comment:
Possibly use a client template here?Not been done before, but you should be able to render placeholders.